pom.xml

```diff
@@ -534,12 +534,12 @@
         <groupId>io.grpc</groupId>
         <artifactId>grpc-protobuf-lite</artifactId>
```

Comment:
This is just setting the version for it via dependency management, but AFAICT no because the `grpc-protobuf` library depends on it. In practice, `grpc-protofbuf` depends on `grpc-protobuf-lite` but excludes the transitive `protobuf-lite` dependency.